PowerShellTools/DebugEngine/DebugServiceEventsHandlerProxy.cs

```diff
@@ -153,7 +153,7 @@ public PSCredential ReadSecureStringPrompt(string message, string name)
         /// <param name="fullName">Full name of remote file(mapped into local)</param>
         public void OpenRemoteFile(string fullName)
         {
-            Debugger.OpenRemoteFile(fullName);
+            Debugger.OpenFileInVS(fullName, true);

```

Comment:
Am going to make the two error messages constants and just have that be passed in instead.